test/factories/task_factory.rb

```diff
@@ -6,11 +6,11 @@
     result 'success'
 
     transient do
-      set_owner nil
+      lock_owner nil
     end
 
     after(:create) do |task, evaluator|
-      ForemanTasks::Lock.owner!(evaluator.set_owner, task.id) if evaluator.set_owner
+      ForemanTasks::Lock.owner!(evaluator.lock_owner, task.id) if evaluator.lock_owner
```

Comment:
Is the lock_owner used anywhere? If not, shouldn't we just remove it?

Reply:
I briefly grep'd through the code and some plugins and couldn't find it (or the original variant) being uesd anywhere. Dropped